ext/drreg/drreg.c

```diff
@@ -1976,7 +1976,10 @@ is_aflags_spill(instr_t *inst)
 #if defined(X86)
```

Comment:
Can you confirm whether drreg-test is fully enabled on ARM? Our Github CI doesn't run the ARM tests (only the cross-compile I think), so maybe run it locally and add to the PR description that it (hopefully) passes or if more work is remaining.

Reply:
I'm still seeing failures of `client.drreg-test` and `client.drreg-end-restore`. I've updated the PR description.